common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationActivityFactory.java

```diff
@@ -77,7 +79,35 @@ public static Intent getAuthorizationActivityIntent(final Context context,
                                                         final AuthorizationAgent authorizationAgent,
                                                         final boolean webViewZoomEnabled,
                                                         final boolean webViewZoomControlsEnabled) {
-        Intent intent;
+        return getAuthorizationActivityIntent(context, authIntent, requestUrl, redirectUri, requestHeaders, authorizationAgent, webViewZoomEnabled, webViewZoomControlsEnabled, null, null);
+    }
+
+    /**
+     * Return the correct authorization activity based on library configuration.
+     *
+     * @param context                    Android application context
+     * @param authIntent                 Android intent used by the authorization activity to launch the specific implementation of authorization (BROWSER, EMBEDDED)
```

Comment:
Does eSTS capture this information today, if I am using MSAL (no broker) and the authorization intent is BROWSER? Is that data present in eSTS telemetry today?

Reply:
ESTS knows about Broker or not today and also knowns about authorization agent.